meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -18,45 +18,38 @@ DESCRIPTION = "Linux container runtime \
  subtle and/or glaring issues. \
  "
 
-SRCREV = "670a205440471d72528f123ab7b686179d04e8e2"
-SRCBRANCH = "17.03.1-resin"
+inherit binary-compress
+FILES_COMPRESS = "/bootstrap/init"
+
+# XXX this should be a temporary branch until we will include mobynit support
+# in a release branch
+SRCREV = "83ea6068d2f94dee4d04078c361e541dc08970b9"
+SRCBRANCH = "petrosagg-wip"
 SRC_URI = "\
   git://github.com/resin-os/docker.git;branch=${SRCBRANCH};nobranch=1 \
   file://docker.service \
+  file://docker-host.service \
   file://var-lib-docker.mount \
   file://docker.conf.systemd \
-  file://0001-graph-aufs-durably-write-layer-on-disk-before-return.patch \
-  file://0002-pkg-ioutils-sync-parent-directory-too.patch \
-  file://0003-pkg-fadvise-implementation-of-posix_fadvise-2.patch \
-  file://0004-pkg-archive-use-fadvise-to-prevent-pagecache-thrashi.patch \
-  file://0005-daemon-cleanup-as-early-as-possible.patch \
-  file://0006-container-make-sure-config-on-disk-has-a-valid-Confi.patch \
 "
 
 # Apache-2.0 for docker
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=aadc30f9c14d876ded7bedc0afd2d3d7"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=9740d093a080530b5c5c6573df9af45a"
 
 S = "${WORKDIR}/git"
 
-DOCKER_VERSION = "17.03.1-ce"
-PV = "${DOCKER_VERSION}+git${SRCREV}"
+BBCLASSEXTEND = " native"
 
-DEPENDS = " \
-  go-cross \
-  btrfs-tools \
-  git \
-  systemd \
-  "
+DOCKER_VERSION = "17.06.0-dev"
+PV = "${DOCKER_VERSION}+git${SRCREV}"
 
-DEPENDS_append_class-target = "lvm2"
-RDEPENDS_${PN} = "curl util-linux iptables tini systemd"
-RRECOMMENDS_${PN} += " kernel-module-dm-thin-pool kernel-module-nf-nat"
+DEPENDS_append_class-native = " go"
+DEPENDS_append_class-target = " systemd go-cross"
+RDEPENDS_${PN}_class-target = " curl util-linux iptables tini systemd"
```

Comment:
Why was this changed from? :

RDEPENDS_${PN} = "curl util-linux iptables tini systemd"

(I guess this is related again about the use of the BBCLASSEXTEND = "native" discussion here for the docker recipe)